File: hack/problems/11/submit.py
      
def add(a,b):
    # write your code here
    return a + b
if __name__=='__main__':
    ls = [int(x) for x in input().split()]
    j = 0
    t = ls[j]
    for i in range(t):
        a = ls[j+1]
        b = ls[j+2]
        j = j+2
        result = add(a,b)
        print(result)

    

File: hack/problems/20/submit.cpp
      
#include<bits/stdc++.h>
using namespace std;

int Sum(int x){
    /* Write your code here */
    int s = 0;
while(x !=0){
int digit = x%10;
s +=digit;
x = x / 10;
}
return s;
}
int main(int argc, char const *argv[])
{
    int t;
    int x;
    int result;
    cin>>t;

    for(int i=0;i<t;i++){
    cin>>x;
    result = Sum(x); 
    cout<<result<<endl;
    }
 	
 	
 	return 0;
} 

    